### ISSUE TITLE
Does the "Development Stage" content need to appear in both Readme and Changelog?

### ISSUE THREAD

**Issue by adam-little** [cagov/design-system#607]

**Comment by adam-little** [comment]
If we think it should appear on both pages, then I suggest moving it above the page navigation tabs

**Comment by m-sullivan7** [comment]
Responding to @adam-little's request on #606, I think it makes sense to omit the development stage tracker from the changelog and use cases tabs/pages.

That said, I'd like a developer opinion on whether seeing the status of a component is useful on a changelog or use case page. I think they'd be the most likely to use that information. @jaredwyee I'll let you decide which/how many developer(s) to ask this question of.

**Comment by xjensen** [comment]
I'm assuming we would want to ditch the tracker box but **keep** the document tabs (on Changelog and Use cases pages)?

**Comment by m-sullivan7** [comment]
Yes, definitely keep the tabs to enable navigation between the pages no matter what. (At least that's my opinion!)

**Comment by chachasikes** [comment]
Is this resolved now?

**Comment by xjensen** [comment]
This is still unresolved. I think we were waiting for a decision regarding which component pages should have the dev stage tracker boxes.

**Comment by adam-little** [comment]
I vote for Readme only. It's the default page so most users will see it.

But I could also see the argument for having it on Changelog since that content is related to the evolution/progress of a component.
